templates/test/ci/prow-clusterclass-ci-default/patches.yaml

```diff
@@ -1,8 +1,46 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
-  name: ci-default
+  name: ${CLUSTER_NAME}
```

Comment:
I think the idea is that a single ClusterClass can be reused by many clusters, so even though this technically works, I don't know that it really demonstrates how a ClusterClass should be used by tying it's name to the cluster's name, which implies there would be one ClusterClass per cluster.

Reply:
That can definitely be misleading.. I'll refactor the name to not be dependent on the cluster name